src/crsAdapter/BewotecExpertAdapter.js

```diff
@@ -791,15 +807,16 @@ class BewotecExpertAdapter {
 
         let travellerAllocationNumber = void 0;
 
-        service.children.forEach((child) => {
+        service.travellers.forEach((ServiceTraveller) => {
+            const traveller = this.helper.traveller.normalizeTraveller(ServiceTraveller, SERVICE_TYPES.hotel);
             let travellerIndex = this.getNextEmptyTravellerLineIndex(crsObject);
             let travellerNumber = CONFIG.crs.lineNumberMap[travellerIndex];
 
             travellerAllocationNumber = travellerIndex + 1;
 
-            crsObject['ta' + travellerNumber] = CONFIG.crs.gender2SalutationMap.child;
-            crsObject['tn' + travellerNumber] = child.name;
-            crsObject['te' + travellerNumber] = child.age;
+            crsObject['ta' + travellerNumber] = traveller.salutation;
```

Comment:
hm, the traveller has no property `.salutation` - it has `.gender` and this has to be mapped to the correct salutation value of the CRS.

Reply:
It is done exactly like in 'assignRoundTripTravellers'. Traveller has this property after 'normalizeTraveller' function finish with it.

Reply:
ah correct - i see. sorry for that.